### PR TITLE
Added code intellisence comments

### PR DESCRIPTION
**Pull request by chalith**

### CODE REVIEW

File: src/clients/base-evernode-client.js [hunk 1]
```diff
@@ -40,18 +40,37 @@ class BaseEvernodeClient {
...
+    * @param {function(event)} handler Callback function to handle the event.
+    */
     once(event, handler) {
         this.events.once(event, handler);
     }
 
+    /**
+     * Deattach the listner event.
+     * @param {string} event Event name.
+     * @param {function(event)} handler (optional) Can be sent if a specific handler need to be deattached. All the handlers will be de attached if not specified.
```
Comment (kithminisg) [inline src/clients/base-evernode-client.js:64]: deattached , de attached => detached

File: src/clients/base-evernode-client.js [hunk 2]
```diff
@@ -103,6 +139,11 @@ class BaseEvernodeClient {
         return [];
     }
 
+    /**
+     * Get the moment from the given XRPL index. (1 Moment - 1190 XRP ledgers).
+     * @param {number} ledgerIndex Ledger index to get the moment value.
+     * @returns Returns the moment of the given XPR ledger index as 'number'. Returns current moment if ledger index is not given.
```
Comment (kithminisg) [inline src/clients/base-evernode-client.js:145]: as a 'number'

File: src/clients/base-evernode-client.js [hunk 3]
```diff
@@ -410,7 +468,13 @@ class BaseEvernodeClient {
         return null;
     }
 
-    // To fetch records from host collection in Firestore database. (Pagination is applied - default page size :20)
+    /**
+     * Get all the hosts registered in Evernode in paginated manar. The result's are paginated. Default page size is 20. Note: Specifing both filter and pagination does not supported.
```
Comment (kithminisg) [inline src/clients/base-evernode-client.js:472]: registered hosts
This returns a  paginated result. (pagination has been mentioned twice)

File: src/clients/base-evernode-client.js [hunk 4]
```diff
@@ -423,7 +487,10 @@ class BaseEvernodeClient {
         return hosts;
     }
 
-    // To fetch all records from config collection in Firestore database.
+    /**
+     * Get all the evernode configuarations without paginating.
```
Comment (kithminisg) [inline src/clients/base-evernode-client.js:491]: all Evernode

Comment (wojake) [reply src/clients/base-evernode-client.js:491]: configurations*

File: src/clients/base-evernode-client.js (hunk 3 above)
Comment (wojake) [reply src/clients/base-evernode-client.js:472]: manner*

File: src/clients/base-evernode-client.js (hunk 2 above)
Comment (wojake) [reply src/clients/base-evernode-client.js:145]: XRP ledger*

**Pull request merged**